src/libs/actions/Policy.js

```diff
@@ -147,7 +147,7 @@ function deleteWorkspace(policyID, reports, policyName) {
 /**
  * Is the user an admin of a free policy (aka workspace)?
  *
- * @param {Array} policies
+ * @param {Record<string, Policy>} policies
```

Comment:
Remove from here and add to function param type

Reply:
@kubabutkiewicz Seems like he is just improving the types of this function that is being used in the actual file he is migrating. This file is still JS

Reply:
ohh right :D didnt see that , thanks @thiagobrez